src/components/tier.jsx

```diff
@@ -61,9 +64,9 @@ class Tier extends React.Component {
 
     return (
       <div>
-        <div className="tier">
+        <div key="this.props.tierId" className="tier">
```

Comment:
I think what you want is `${this.props.tierId}`.
Double quotes without curly brackets probably won't give you what you're looking for

Reply:
Oops, thank you.